www/pfSense-pkg-squid/files/usr/local/www/widgets/widgets/squid_antivirus_status.widget.php

```diff
@@ -159,3 +161,26 @@ function squid_antivirus_statistics() {
 	</tbody>
 	</table>
 </div>
+
+<script type="text/javascript">
+//<![CDATA[
+function getstatus_squidav() {
+	$.ajax({
+		type: 'get',
+		url: '/widgets/widgets/squid_antivirus_status.widget.php',
+		dataType: 'html',
+		dataFilter: function(raw){
+			// We reload the entire widget, strip this block of javascript from it
+			return raw.replace(/<script>([\s\S]*)<\/script>/gi, '');
+		},
+		success: function(data){
+			$('#squidav_status').html(data);
+		}
+	});
```

Comment:
It's good practice to include an error handler. In this case it could return an HTML error message?